src/beginner-tips/assets/tabs/QuickStartPanel.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import React from 'react';
+
+const isMac: boolean = navigator.platform.toLowerCase().indexOf("darwin") === 0;
+
+export default function QuickStartPanel() {
+  const codeSample = (
+    <code>
```

Comment:
what's the font of this `code` tag?

Reply:
Seems a default one provided by VS Code's webview.

Reply:
it was overridden as `monospace` by agent. will update